pages/partials/title.ejs

```diff
@@ -60,6 +60,7 @@
     }
 %>
 <title>
+    <% if (config.devMode) { %>[DEVMODE]<% } %>    
```

Comment:
```suggestion
    <% if (config.devMode) { %>[DEV]<% } %>    
```
Or anything shorter, just to preserve as much space as possible for the real title content.